ocs_ci/utility/utils.py

```diff
@@ -905,6 +909,41 @@ def get_cluster_version_info():
     return cluster_version_info
 
 
+def get_ocs_operator_version():
+    """
+    Gets the ocs operator version info
+
+    Returns:
+        str: ocs operator version
+
+    """
+    from ocs_ci.ocs.ocp import OCP
+    ocp = OCP(kind="catalogsource", namespace="openshift-marketplace")
+    for item in ocp.get().get('items'):
+        if item.get('metadata').get('name') == "ocs-catalogsource":
+            return item.get('spec').get('image')
+
+
+def get_ocs_build_number():
+    """
+    Gets the build number for ocs operator
+
+    Return:
+        str: build number for ocs operator version
```

Comment:
I think that this function should return just build ID without any text `BUILD ID:` .
The `BUILD ID:` should be used in the place where you will use it.  Line: https://github.com/red-hat-storage/ocs-ci/pull/853/files#diff-68bc5ed905b5a4cefbac287573f4f60fR875

or somewhere above you can have:
So it can look like: 
```python
build_id = get_ocs_build_number()
build_id_msg = f"BUILD ID: {build_id} " if build_id else ''

# then instead your line you can have:
f"({build_id_msg}"
```

Reply:
Done